deployment/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM ubuntu:xenial
 # From: https://github.com/nodejs/docker-node/blob/master/6.11/Dockerfile
 
 RUN apt-get update && apt-get install -y --no-install-recommends \
-        gnupg curl ca-certificates xz-utils wget libgconf-2-4 \
+        gnupg curl ca-certificates xz-utils wget libgtk2.0-0 libgconf-2-4 \
```

Comment:
So if I understand correctly, `docker-build-and-push` has been broken since we changed the repo's name to `orca`. This [dummy](https://github.com/plotly/orca/compare/dummy) branch [seems](https://circleci.com/workflow-run/8712231e-8cde-4da6-a767-0647a011db94) to confirm the issue. 

Now, why does adding `libgtk2.0` fix this issue?

Reply:
I'm wondering this too. (As a future note, it would be great to put an explanation in the commit comment.)